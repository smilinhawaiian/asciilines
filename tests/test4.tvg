9 9
* 9 0 h 9
9 2 1 v 5
# 5 2 h 9
x -1 5 v 9
