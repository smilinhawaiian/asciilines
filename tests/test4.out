.....x...
.....x...
.9...x...
.9...x...
.9...x...
.9###x###
.9...x...
.9...x...
*9***x***
